apps/opencs/model/world/columnbase.hpp

```diff
@@ -121,14 +131,32 @@ namespace CSMWorld
 
         virtual std::string getTitle() const;
 
-        virtual int getId() const;
+        virtual int getId() const; // FIXME: why have an accessor for a public member?
+    };
+
+    class NestableColumn : public ColumnBase
+    {
+        std::vector<NestableColumn *> mNestedColumns;
+        bool mHasChildren;
```

Comment:
Is mHasChildren needed? Seems to me you might be able to just check if mNestedColumns.empty().